composer.json

```diff
@@ -44,7 +44,8 @@
     "facebook/graph-sdk": "^5.7",
     "abraham/twitteroauth": "^4.0",
     "codeinwp/themeisle-sdk": "^3.3",
-    "vkcom/vk-php-sdk": "5.101.0"
+    "vkcom/vk-php-sdk": "5.101.0",
+    "irazasyed/telegram-bot-sdk": "^3.9"
```

Comment:
I think it is better to pin down the version to 3.9.0 to prevent any updates from depandabot since 3.9.1 requires 8.0 and our supported version must be 7.4

Reply:
@Soare-Robert-Daniel Done! Thank you for the review!